### PR TITLE
Failing query range request due to top level operation dependency

### PR DESCRIPTION
**Pull request by Rajat-Dabade**
This PR fixes the query range bad request issue for the service overview latency graph.

### CODE REVIEW

File: frontend/src/container/MetricsApplication/Tabs/Overview/ServiceOverview.tsx [hunk 1]
```diff
@@ -49,6 +49,8 @@ function ServiceOverview({
 		[servicename, tagFilterItems, isSpanMetricEnable, topLevelOperationsRoute],
 	);
 
+	const isQueryEnabled = topLevelOperationsRoute.length > 0;
```
Comment (palashgdev) [inline frontend/src/container/MetricsApplication/Tabs/Overview/ServiceOverview.tsx:52]: i like it thanks @Rajat-Dabade 👍

File: frontend/src/container/GridGraphLayout/Graph/index.tsx [hunk 2]
```diff
@@ -287,7 +288,10 @@ function GridCardGraph({
 		);
 	}
 
-	if (prevChartDataSetRef?.labels === undefined && queryResponse.isLoading) {
+	if (
+		prevChartDataSetRef?.labels === undefined &&
```
Comment (palashgdev) [inline frontend/src/container/GridGraphLayout/Graph/index.tsx:292]: can you confirm do we need `prevChartDataSetRef` check also ?

Comment (Rajat-Dabade) [reply frontend/src/container/GridGraphLayout/Graph/index.tsx:292]: Got it, we don't need this one as it is handled down below.

Comment (Rajat-Dabade) [reply frontend/src/container/GridGraphLayout/Graph/index.tsx:292]: I will remove it.

**Review by palashgdev — APPROVED** [review]

**Pull request merged**